client/components/forms/multi-checkbox/index.jsx

```diff
@@ -43,24 +37,28 @@ var MultiCheckbox = module.exports = React.createClass({
 		} );
 
 		event.stopPropagation();
-	},
-
-	getCheckboxElements: function() {
-		var checked = this.props.checked || this.state.initialChecked;
-
-		return this.props.options.map( function( option ) {
-			var isChecked = checked.indexOf( option.value ) !== -1;
-
-			return (
-				<label key={ option.value }>
-					<input name={ this.props.name + '[]' } type="checkbox" value={ option.value } checked={ isChecked } onChange={ this.handleChange } disabled={ this.props.disabled } />
-					<span>{ option.label }</span>
-				</label>
-			);
-		}, this );
-	},
+	};
 
-	render: function() {
-		return <div className="form-checkbox-group" { ...omit( this.props, Object.keys( MultiCheckbox.propTypes ) ) }>{ this.getCheckboxElements() }</div>;
+	render() {
+		const { disabled, name, options } = this.props;
+		const checked = this.props.checked || this.state.initialChecked;
+		return (
+			<div className="multi-checkbox" { ...omit( this.props, Object.keys( MultiCheckbox.propTypes ) ) }>
+				{ options.map( ( option ) => (
+						<label key={ option.value }>
```

Comment:
I think a general rule of thumb is that a line should only be indented a maximum of one from its previous line, with the exception of multi-line `if` conditions ([for which there is a guideline](https://github.com/Automattic/wp-calypso/blob/master/docs/coding-guidelines/javascript.md#multi-line-statements)). In this case, I think you'd have two alternatives:

```js
{ options.map( ( option ) => (
	<label key={ option.value }>
		<input
			name={ name + '[]' }
			type="checkbox"
			value={ option.value }
			checked={ includes( checked, option.value ) }
			onChange={ this.handleChange }
			disabled={ disabled }
		/>
		<span>{ option.label }</span>
	</label>
) ) }
```

```js
{
	options.map( ( option ) => (
		<label key={ option.value }>
			<input
				name={ name + '[]' }
				type="checkbox"
				value={ option.value }
				checked={ includes( checked, option.value ) }
				onChange={ this.handleChange }
				disabled={ disabled }
			/>
			<span>{ option.label }</span>
		</label>
	) )
}
```

My preference being toward the first.